build/build-extensions.ps1

```diff
@@ -135,48 +133,57 @@ function CreateSiteExtensions() {
     Copy-Item -Path $buildOutput\publish\win-x86\ -Destination $officialSiteExtensionPath\32bit -Force -Recurse > $null
     Copy-Item -Path $buildOutput\publish\win-x64 -Destination $officialSiteExtensionPath\64bit -Force -Recurse > $null
     Copy-Item -Path $officialSiteExtensionPath\32bit\applicationHost.xdt -Destination $officialSiteExtensionPath -Force > $null
-
-    Write-Host "======================================"
-    Write-Host "Deleting workers directory: $officialSiteExtensionPath\32bit\workers" 
+    Write-Host "  Deleting workers directory: $officialSiteExtensionPath\32bit\workers" 
     Remove-Item -Recurse -Force "$officialSiteExtensionPath\32bit\workers" -ErrorAction SilentlyContinue
-    Write-Host "Moving workers directory:$officialSiteExtensionPath\64bit\workers to" $privateSiteExtensionPath 
-    Move-Item -Path "$officialSiteExtensionPath\64bit\workers"  -Destination "$officialSiteExtensionPath\workers" 
+    Write-Host "  Moving workers directory: $officialSiteExtensionPath\64bit\workers to $officialSiteExtensionPath\workers"
+    Move-Item -Path "$officialSiteExtensionPath\64bit\workers" -Destination "$officialSiteExtensionPath\workers" 
      
     # This goes in the root dir
-    Copy-Item $rootDir\src\WebJobs.Script.WebHost\extension.xml $siteExtensionPath > $null
-    
+    Copy-Item $rootDir\src\WebJobs.Script.WebHost\extension.xml $siteExtensionPath > $null    
     
     Write-Host "Done copying. Elapsed: $($stopwatch.Elapsed)"
     Write-Host "======================================"
     Write-Host ""
 
-    Write-Host "Generating $hashesForHardlinksFile"
     Write-Host "======================================"
+    Write-Host "Generating $hashesForHardlinksFile"    
     WriteHashesFile $siteExtensionPath/$extensionVersionNoSuffix
-    Write-Host "Done generating $hashesForHardlinksFile"
+    Write-Host "Done generating $siteExtensionPath/$extensionVersionNoSuffix"
```

Comment:
Was this intentional?

Reply:
Yeah, i like logging the full path rather than just the file name like it was previously

Reply:
Got it.  But 149 should be changed too then so it matches "Generating .... Done generating".
Suggest mentioning the context in the log message as the path alone does not communicate what is happening.
Either
Write-Host "Generating $siteExtensionPath/$extensionVersionNoSuffix/$hashesForHardlinksFile"
...
Write-Host "Done generating $siteExtensionPath/$extensionVersionNoSuffix/$hashesForHardlinksFile"

OR 

Write-Host "Generating hashes for hard links"
...
Write-Host "Done generating hashes for hard links into "$siteExtensionPath/$extensionVersionNoSuffix"

Reply:
i didn't see that on 149 -- you're right. Let me update!